benchmark/haskell/ac-bench.cabal

```diff
@@ -0,0 +1,45 @@
+cabal-version: 1.12
+
+-- This file has been generated from package.yaml by hpack version 0.34.6.
+--
+-- see: https://github.com/sol/hpack
```

Comment:
Since we're using  hpack which has a `package.yaml` instead, we can `.gitignore` this file since it's generated.